paymentsheet/src/main/java/com/stripe/android/paymentsheet/FlowControllerCompose.kt

```diff
@@ -163,6 +163,7 @@ private fun internalRememberPaymentSheetFlowController(
             statusBarColor = { activity.window?.statusBarColor },
             paymentOptionCallback = paymentOptionCallback,
             paymentResultCallback = paymentResultCallback,
+            initializedViaCompose = true,
```

Comment:
Should this not also be added for `rememberPaymentSheet`?

Reply:
Good catch. I did most of the work, but missed the area that calls it!